src/content/public-goods-funding/index.md

```diff
@@ -0,0 +1,289 @@
+
+Unlike [private goods](https://www.investopedia.com/terms/p/private-good.asp), public goods provide benefits for the majority, not just one individual. Such goods are necessary in creating a future that results in positive outcomes for all members of society. Public roads, for example, facilitate commerce and increase societal prosperity and wellbeing.
+
+Traditionally, public goods are usually made available by governments (through taxes collected from citizens). Private markets can also create public goods: the radio is an example of a public good—broadcast stations cannot prevent listeners from receiving signals (i.e., it is nonexcludable) and multiple individuals can listen at the same time (i.e., it is nonrivalrous).
+
+However, the conventional models of building and funding public goods run into various problems. Not only have these issues stymied the development of public goods, but it has also resulted in public goods with [negative externalities](https://corporatefinanceinstitute.com/resources/knowledge/economics/negative-externalities/).
+
+### Problems of traditional public goods funding {problems-of-traditional-public-goods-funding}
+
+### 1. Misaligned incentives {#misaligned-incentives}
+
+Traditional methods of funding public goods suffer from misaligned incentives among different entities coordinating to allocate resources. These incentive problems can be described using several concepts from economic theory:
+
+#### The free-rider problem {#free-rider-problem}
+
+The [free-rider problem](https://bit.ly/3ATE3KC) describes a situation whereby some individuals refuse to pay towards building a public good, but intend to use it. In this case, such individuals are "taking a free ride" on the contributions of others towards funding the public good.
+
+The free-rider problem is central to the problem of funding public goods. If everyone refuses to pay for a public good, hoping someone else does, it becomes impossible to fund and build the public good.
+
+#### Tragedy of commons {#tragedy-of-commons}
+
+The [tragedy of commons](https://www.investopedia.com/terms/t/tragedy-of-the-commons.asp) concerns the degradation of commonly shared resources through overconsumption. A common resource is similar to a public good (they are nonexcludable), but can typically diminish in quality and availability as more people use it (i.e., they are rivalrous).
+
+Because individuals cannot prevent others from consuming the common resource, they are incentivized to take from the shared resource pool without replenishing it. But, as more people consume the resource, either of two things can happen: a) supply of the resource dwindles until there isn't any left for public consumption b) the quality of the resource degrades over time until it becomes unusable. Both situations are non-ideal, which is why solving the tragedy of commons is a key goal of new designs for funding public goods.
```

Comment:
```suggestion
Because individuals cannot prevent others from consuming the common resource, they are incentivized to take from the shared resource pool without replenishing it. But, as more people consume the resource, either of two things can happen: a) the supply of the resource reduces until there isn't any left for public consumption b) the quality of the resource degrades over time until it becomes unusable. Both situations aren't ideal, which is why solving the tragedy of commons is a key goal of new designs for funding public goods.
```